cql/DashboardLibrary.cql

```diff
@@ -249,6 +251,7 @@ define function BiopsyInterpretation(conclusions List<System.Concept>):
         when c in "Histologic CIN3" then {text: 'CIN3', rank: 6}
         when c in "AIS" then {text: 'AIS', rank: 7}
         when c in "Histologic cancer" then {text: 'Cancer', rank: 8}
+        when c ~ "Insufficient" then {text: 'Insufficient/Unsatisfactory', rank: 1}
```

Comment:
Can we add a test to make sure that somebody with an "Insufficient" result would be showing up correctly? Maybe we can add a test case to [test/DisplayCervicalCancerMedicalHistory](https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/tree/CCSMCDS-284-add-insufficient-histology/test/DisplayCervicalCancerMedicalHistory/cases) to check that [HistologyDiagnosticReportsSummary](https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/blob/main/cql/DashboardLibrary.cql#L978-L979) is displaying correctly?

Reply:
I created a testcase and got issues. Can you take a look at https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/154/commits/b17626bab1f69d4cb30de1c9ea3cff8b40cec3bb?

My problems are:

1. Given the testcase, I expected to see `value` `grade` `label` in the result json, but I couldn't. I wonder if there's something missing in the testcase.
2. Since I'm not familiar with CQL test syntax, what would be a method to extract text from `Interpretation` from FormatReport, rather than comparing the result to the whole json (e.g. {"value":"Insufficient"})

Reply:
Good catch. I do not think that the CQL Testing Framework has a way at the time to compare individual elements within a greater CQL Expression result. Over Slack, I proposed two ways that we may be able to get around this for the time being.

Reply:
Added suggested testscript https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/154/commits/eb470ced8ff011e2304032de07459e2e67f778ed

Reply:
New test looks great, nice job setting it up. This way, we're testing that our CQL is actually able to read-in and process one of these Insufficient results. I added a commit just to update the name of the test on line 2: https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/154/commits/5d861d2d4bba4d57064db212f4a79233cba53a9d.